examples/with-lokijs/nuxt.config.js

```diff
@@ -0,0 +1,80 @@
+// import session from "express-session"
+import bodyParser from "body-parser"
+const pkg = require("./package")
+
+module.exports = {
+  mode: "universal",
+
+  /*
+   ** Headers of the page
+   */
+  head: {
+    title: pkg.name,
+    meta: [
+      { charset: "utf-8" },
+      { name: "viewport", content: "width=device-width, initial-scale=1" },
+      { hid: "description", name: "description", content: pkg.description }
+    ],
+    link: [{ rel: "icon", type: "image/x-icon", href: "/favicon.ico" }]
+  },
+
+  /*
+   ** Customize the progress-bar color
+   */
+  loading: { color: "green" },
+
+  /*
+   ** Add server middleware
+   ** Nuxt.js uses `connect` module as server
+   ** So most of express middleware works with nuxt.js server middleware
+   */
+  serverMiddleware: [
+    // body-parser middleware
+    bodyParser.json(),
```

Comment:
Using bodyParser for all requests slows down SSR and makes problem with Proxy. So i suggest moving it to express middleware stack.

Reply:
very good point. Thanks